src/components/skinManager.js

```diff
@@ -34,15 +34,12 @@ define(['apphost', 'userSettings', 'browser', 'events', 'pluginManager', 'backdr
             id: "dark",
             isDefault: true,
             isDefaultServerDashboard: true
-        }, {
-            name: "Emby",
-            id: "emby"
```

Comment:
why not rename to `emerald`?

Reply:
This theme is almost exactly the same as the dark theme and maintaining them both will be a pain until we have better theme support. Working on custom themes is a better alternative to leaving this in the web source and having to deal with it.